### ISSUE TITLE
Incorrect heading structure on Form pages if "When to use this form" section is not displayed

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14652]
## Description
On VA forms pages, there is an options section that can be added in Drupal for "When to use this form". When this section is added, an H2 heading is displayed on the front end, when this section is not added, this H2 heading is not present. This creates a heading structure of H1 > H3 and a skipped heading violation.

## Examples

- Page with proper heading structure: [About VA Form 21-4142](https://www.va.gov/find-forms/about-form-21-4142/)
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/a71db209-625d-4781-a23f-dc61b582cd48)

- Page with skipped heading: [About VA Form 10-0491J](https://www.va.gov/find-forms/about-form-10-0491j/)
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/cb6ee7b6-ba76-419f-a75e-e06284336e58)

## Acceptance Criteria
- [x] Heading structure does not skip levels regardless of which content sections are present on the page.
- [ ] Accessibility review

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

**Comment by laflannery** [comment]
Approved!

**Comment by randimays** [comment]
Verified successfully in production. Closing!
